src/scriptworker/utils.py

```diff
@@ -258,7 +259,9 @@ async def retry_async(
     while True:
         try:
             return await func(*args, **kwargs)
-        except retry_exceptions:
+        except retry_exceptions as exc:
+            if log_exceptions:
+                log.info(f"retry_async exception:\n{type(exc)} {exc}")
```

Comment:
Nit: Should we `log.warning()` instead? I get the feeling an exception is worth being called out as a warning.

Reply:
Hm. Today, we retry on exception, and we don't log anything if we're retrying; we silently eat the error and sleep, so just logging anything is an improvement.

I don't have a strong objection to warning though, so I can try that.